api/v1alpha1/managedfleetnotificationrecord_types.go

```diff
@@ -137,8 +141,8 @@ func (fnr *ManagedFleetNotificationRecord) HasNotificationRecordItem(mc, name, c
 	return false
 }
 
-// CanBeSent checks if the service log for the notification can be sent for the given hosted cluster
-func (fnr *ManagedFleetNotificationRecord) CanBeSent(mc, name, clusterID string) (bool, error) {
+// FiringCanBeSent checks if the notification can be sent for a firing alert for the given hosted cluster
+func (fnr *ManagedFleetNotificationRecord) FiringCanBeSent(mc, name, clusterID string) (bool, error) {
```

Comment:
Renamed this to be a specific check for firing, we don't need to check anything when handling a resolved event with the current ocm-agent implementation. In the future, we might want to add one to handle resolving service log sends.